proto/validator/accounts/v2/web_api.proto

```diff
@@ -0,0 +1,100 @@
+syntax = "proto3";
+package ethereum.validator.accounts.v2;
+
+import "eth/v1alpha1/attestation.proto";
+import "eth/v1alpha1/beacon_block.proto";
+import "google/api/annotations.proto";
+import "google/protobuf/empty.proto";
+
+service Wallet {
+    rpc CreateWallet(CreateWalletRequest) returns (WalletResponse) {
+        option (google.api.http) = {
+            post: "/v2/validator/wallet/create"
+        };
+    }
+    rpc EditConfig(EditWalletConfigRequest) returns (WalletResponse) {
+        option (google.api.http) = {
+            post: "/v2/validator/wallet/config/edit"
+        };
+    }
+    rpc WalletConfig(google.protobuf.Empty) returns (WalletResponse) {
+        option (google.api.http) = {
+            get: "/v2/validator/wallet/config"
+        };
+    }
+}
+
+service Accounts {
+    rpc CreateAccount(google.protobuf.Empty) returns (CreateAccountResponse) {
+        option (google.api.http) = {
+            post: "/v2/validator/accounts/create"
+        };
+    }
+    rpc ListAccounts(ListAccountsRequest) returns (ListAccountsResponse) {
+        option (google.api.http) = {
+            get: "/v2/validator/accounts"
+        };
+    }
+}
+
+message CreateWalletRequest {
+    // Path on disk where the wallet will be stored.
+    string wallet_path = 1;
+    // Type of key manager for the wallet, either direct, derived, or remote.
+    enum KeyManagerKind {
```

Comment:
KeymanagerKind* to match the format elsewhere in the repo. Also, should UNKNOWN be 0? Maybe we should try to match the actual enum we use in validator/keymanager/v2